packages/babel-plugin-transform-es2015-block-scoping/README.md

```diff
@@ -14,12 +14,24 @@ npm install --save-dev babel-plugin-transform-es2015-block-scoping
 
 **.babelrc**
 
+Without options:
```

Comment:
I was just copying https://github.com/babel/babel/pull/5194/files like I was asked. :) Updated and cut the error message down a bit.